CHANGELOG.md

```diff
@@ -6,6 +6,25 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [7.0.0]
+### Added
+- Integrate `PollingController` mixin into `SmartTransactionsController` and implement the abstract `_executePoll` method ([#237](https://github.com/MetaMask/smart-transactions-controller/pull/237))
+- Validation can be now be circumvented by passing the `skipConfirm` option ([#271](https://github.com/MetaMask/smart-transactions-controller/pull/271))
+
+### Changed
+- **BREAKING**: Bump `@metamask/network-controller` from `^15.0.0` to `^17.0.0` ([#238](https://github.com/MetaMask/smart-transactions-controller/pull/238) [#241](https://github.com/MetaMask/smart-transactions-controller/pull/241))
+  - This is breaking because the type of the `messenger` has backward-incompatible changes. See the changelog for `@metamask/base-controller@4.0.0` for more.
+- **BREAKING**: `supportedChainIds` now default default to including Goerli instead of Rinkeby ([#237](https://github.com/MetaMask/smart-transactions-controller/pull/237))
+- **BREAKING**: Minimum Node.js version is now 18.18 ([#270](https://github.com/MetaMask/smart-transactions-controller/pull/270))
+- Add optional options object containing a `networkClientId` argument to `updateSmartTransaction` which, if passed, is used fetch the chainId that corresponds to the networkClientId and is then used to fetch and update the status of pending smart transactions for that chainId ([#237](https://github.com/MetaMask/smart-transactions-controller/pull/237))
+- Bump `@metamask/controller-utils` from `^5.0.0` to `^8.0.3` ([#242](https://github.com/MetaMask/smart-transactions-controller/pull/242) [#244](https://github.com/MetaMask/smart-transactions-controller/pull/244))([#242](https://github.com/MetaMask/smart-transactions-controller/pull/242)) ([#244](https://github.com/MetaMask/smart-transactions-controller/pull/244)) ([#267](https://github.com/MetaMask/smart-transactions-controller/pull/267)) ([#272](https://github.com/MetaMask/smart-transactions-controller/pull/272))
```

Comment:
[Little change](https://github.com/MetaMask/smart-transactions-controller/pull/271/files#diff-e9fc493756187de210ae908875fec1b6b41a3f79945a8f9485f18fdb90489299L577) in 271 that sneaked in:

``` suggestion
- `getFees` no longer needs to take an approval transaction as the second argument ([#271](https://github.com/MetaMask/smart-transactions-controller/pull/271))
- Bump `@metamask/controller-utils` from `^5.0.0` to `^8.0.3` ([#242](https://github.com/MetaMask/smart-transactions-controller/pull/242) [#244](https://github.com/MetaMask/smart-transactions-controller/pull/244))([#242](https://github.com/MetaMask/smart-transactions-controller/pull/242)) ([#244](https://github.com/MetaMask/smart-transactions-controller/pull/244)) ([#267](https://github.com/MetaMask/smart-transactions-controller/pull/267)) ([#272](https://github.com/MetaMask/smart-transactions-controller/pull/272))

Reply:
Applied in [9b8ca7a](https://github.com/MetaMask/smart-transactions-controller/pull/276/commits/9b8ca7a13ffa9404a0cae1781077c250085b0a63).